### DISCUSSION TITLE
Discord Webhook'ları

### DISCUSSION THREAD

**Discussion by erenndev** [community/community#19503] [Actions] [answered]
Merhaba, ben discord webhookları ile ilgili bir soru sormak istiyorum,

- Discord Webhooklarını github depolarıyla hangi yollar ile kullanabilirim?
- Discord Webhooklarını en verimli şekilde kullanabileceğim yer github mıdır?

**Comment by Huseyin-Cinar** [comment, marked as answer]
Eren discord webhooklarını github depolarından kullanabilirsin ama kendin yapabileceğini sanmıyorum bu yüzden;

- Bu tür projeler yapmış insanların depolarını araştır.
- İkinici soruna ise cevabım hayır. Discord Webhooklarını en verimli kullanabileceğin yer bir discord botudur.
- Ama en kolay nasıl kullanabileceğini merak ediyorsan ----> https://discohook.org/ adresine bak.

- Yanıt olarak işaretlemeyi unutma :)
